virocon/distributions.py

```diff
@@ -1347,3 +1348,140 @@ def _fit_mle(self, sample):
 
     def _fit_lsq(self, data, weights):
         raise NotImplementedError()
+ 
+class VonMisesDistribution(Distribution):
+    """
+    A Von Mises (Circular Norm) Distribution. 
+    
+    The distributions probability density function is given by [1]_: 
+    
+    :math:`f(x) = \frac{\exp{(\kappa \cos{(x - \mu)})}}{2 \pi I_0(\kappa)}`
```

Comment:
The backslashes need to be escaped, as they have a special meaning in python strings. 
Thus this should be instead:
`\\frac{\\exp{(\\kappa \\cos{(x - \\mu)})}}{2 \\pi I_0(\\kappa)}`